checks/raw/pinned_dependencies_test.go

```diff
@@ -728,6 +728,24 @@ func TestDockerfileInsecureDownloadsLineNumber(t *testing.T) {
 					endLine:   64,
 					t:         checker.DependencyUseTypePipCommand,
 				},
+				{
+					snippet:   "bash <(curl --silent --show-error \"https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash\")",
```

Comment:
just fyi, this is a good time to use backticks (`` ` ``)so you don't need to escape the quotes which are part of the snippet

Reply:
Thanks for this (and the other) tips. I'm quite the Go n00b. I'll action these tomorrow 👍